user/applications/tinker/src/fqc_test.cpp

```diff
@@ -258,15 +258,18 @@ bool FqcTest::bleScan(JSONValue req) {
 
 // Skip 6 and 7 since they are SWC+SWD (PB3, PA27)
 // Skip 8 and 9 since they are Uart TX+RX (PA7, PA8)
-// The pin order denotes how they are jumpered together. IE pin 14 (PB4) and pin 21 (PB31) are jumpered together.
+// The pin order denotes how they are jumpered together. IE pin A5 (PB4) and pin S6 (PB31) are jumpered together.
 // The pin order largely reflects which pins are physically next to each other on the P2 jig and can be easily jumpered together. 
-// Two sets of pins are not adjacent and require a hookup wire:
-// - Pin19 (PA0) + Pin10 (PA15 ie WKP/D10)
-// - Pin14 (PB4) + Pin21 (PB31)
-static const Vector<uint16_t> p2_digital_pins = {14, 21, 10, 19, 0, 1, 15, 16, 17, 12, 18, 2, 4, 5, 3, 11, 13, 20 };
+// Two sets of pins on the P2 jig are not adjacent and require a hookup wire:
+// - S4 (PA0) + WKP/D10 (PA15)
+// - A5 (PB4) + S6 (PB31)
+static Vector<uint16_t> p2_gpio_test_pins = { A5, S6, WKP, S4, D0, D1, S0, S1, S2, A1, S3, D2, D4, D5, D3, A0, A2, S5 };
+static Vector<uint16_t> photon2_gpio_test_pins = {A0, A1, A2, A5, S4, S3, S2, S0, S1, D0, D1, D2, D3, D4, D5, D10 };
```

Comment:
These pin pairs are a best guess based on the adjacent feather pins. The real test jig may route pins differently and require changes to the pairing. But this is a start for now